api/main.py

```diff
@@ -259,6 +259,199 @@ def settle_btc_payments():
         else:
             time.sleep(10)
 
+@app.cli.command("lightning-payments-processor")
```

Comment:
This is equivalent to settle-btc-payments, but for lightning, no? So let's call it settle-lightning-payments, rather than "payment processor"...